README.md

```diff
@@ -40,8 +40,8 @@ hard-coded defaults and the module parameter values specified on the command lin
 | EVENT_LOG | USER_BLOCK              | {any}                       | If enabled, log user blocked events to the event log (default: enabled=false)       |
```

Comment:
In the slack channel I tried to point out, that the parameters login.fail.attempts, login.fail.to.warn.attempts, and login.fail.timeout can be specified by "code" and not by "configName", compare the method mockHttpCalls() in
https://github.com/folio-org/mod-login/blob/missing_config_in_readme/src/test/java/org/folio/logintest/EventsLoggingTests.java .